draft-linus-trans-gossip-ct.md

```diff
@@ -282,7 +284,7 @@ with the following content:
 
   - x509_chain: An array of base64-encoded X.509 certificates. The
     first element is the end-entity certificate, the second chains to
-    the first and so on.
+    the first and so on. [clients may not receive such chains - should this be a requirement? e.g. they may get a bag of certs that admits multiple chains, they may receive chains with missing certs, user might've clicked through a warning etc]
```

Comment:
I see. I think there is value in sending both the constructed and supplied chains.